releases/evm/2025-02-xrplevm-ITS-v2.1.0.md

```diff
@@ -0,0 +1,127 @@
+# # Ripple EVM Sidechain ITS v2.1.0
+
+|                | **Owner**                                                                   |
+| -------------- | --------------------------------------------------------------------------- |
+| **Created By** | @blockchainguyy <ayush@interoplabs.io>                                      |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io>, @talalashraf <talal@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+[Release](https://github.com/axelarnetwork/interchain-token-service/releases/tag/v)
+
+## Background
+
+- This is the Ripple EVM sidechain ITS release.
+
+## Deployment
+
+Ensure that [Ripple EVM GMP](../evm/2025-02-xrplevm-GMP-v6.4.0.md) is deployed first.
+
+```bash
+# Clone latest main and update deps
+npm ci
+```
+
+Create an `.env` config. Use `all` for `CHAINS` to run the cmd for every EVM chain, or set a specific chain.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAINS=all
+```
+
+### Devnet Amplifier
+
+Amplifier ITS
+
+```bash
+# Deploy new implementation
+node evm/deploy-its.js -s "v1.1.0 devnet-amplifier" -m create2
+```
+
+### Stagenet / Testnet / Mainnet
+
+```bash
+# Deploy new implementation
+node evm/deploy-its.js -s "v2.1.0" -m create2
```

Comment:
proxy salt is missing. Also, since it's a staging connection, add some suffix to both the salts